components/form/Checkbox.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+export function Checkbox({ name, label, value }) {
+  const {
+    register,
+    formState: { errors }
+  } = useFormContext();
+  if (!value && !label) return null;
+  return (
+    <>
+      <div className="flex items-center">
+        <input
+          id={`options-${value}`}
```

Comment:
Lets change this to `checkbox` to make it more explanatory. 

```suggestion
          id={`checkbox-${value}`}
```